internal/ls/utilities.go

```diff
@@ -245,13 +245,99 @@ type PossibleTypeArgumentInfo struct {
 	nTypeArguments int
 }
 
-// !!! signature help
+// Get info for an expression like `f <` that may be the start of type arguments.
 func getPossibleTypeArgumentsInfo(tokenIn *ast.Node, sourceFile *ast.SourceFile) *PossibleTypeArgumentInfo {
-	return nil
-}
+	// This is a rare case, but one that saves on a _lot_ of work if true - if the source file has _no_ `<` character,
+	// then there obviously can't be any type arguments - no expensive brace-matching backwards scanning required
+	// searchPosition := len(sourceFile.Text())
```

Comment:
Is this missing a `// !!!`, or are we getting rid of this code?

Reply:
Yes, getting rid of it.